bzl/deps.bzl

```diff
@@ -0,0 +1,78 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def djinni_deps():
+
+    native.android_sdk_repository(name = "androidsdk")
+    native.android_ndk_repository(name = "androidndk", api_level = 21)
+
+    # Bazel Skylib needed for Scala rules
+    bazel_skylib_version = "1.0.2"
+    http_archive(
```

Comment:
to be good citizens here we should wrap these call in maybe in case the root project already set these deps up...

https://docs.bazel.build/versions/main/repo/utils.html#maybe